app/src/main/java/ru/vssemikoz/newsfeed/DataBase/NewsAppDataBase.java

```diff
@@ -0,0 +1,12 @@
+package ru.vssemikoz.newsfeed.DataBase;
+
+import androidx.room.Database;
+import androidx.room.RoomDatabase;
+
+import ru.vssemikoz.newsfeed.DAO.NewsItemDAO;
+import ru.vssemikoz.newsfeed.models.NewsItem;
+
+@Database(entities = {NewsItem.class}, version = 3, exportSchema = false)
+public abstract class NewsAppDataBase extends RoomDatabase {
+    public abstract NewsItemDAO newsItemDAO();
```

Comment:
Be careful with abbreviations. There's no concrete requirement on how to name them. Both `OrderDAO` and `OrderDao` are correct. Pay attention to the project's code style.